### DISCUSSION TITLE
Why does adding a model change the physics of the original model?

### DISCUSSION THREAD

**Discussion by yangwxb** [sofa-framework/sofa#4718] [Write / Run a simulation] [answered]
Sorry to bother you, but I am now faced with an incomprehensible phenomenon: when I added a model to the original scn file, the sofa effect of the original model changed a lot, and I did not change any of its parameters.

**Comment by epernod** [comment]
Hi,

what do you mean by added a model ?
Are you adding another simulated object in the scene? if yes check the collision 

Otherwise post the scene before and after here.

Best regards

**Reply by yangwxb** [reply, marked as answer]
Thank you for your reply, this problem has been solved, is the solver position under the root node problem.
